--- parts/wrapper/paste/script/command.py	2011-06-17 15:14:30.000000000 +0300
+++ parts/wrapper/paste/script/command.py	2011-06-17 15:13:51.000000000 +0300
@@ -33,13 +33,13 @@
 class NoDefault(object):
     pass
 
-dist = pkg_resources.get_distribution('PasteScript')
+#dist = pkg_resources.get_distribution('PasteScript')
 
 python_version = sys.version.splitlines()[0].strip()
 
 parser = optparse.OptionParser(add_help_option=False,
-                               version='%s from %s (python %s)'
-                               % (dist, dist.location, python_version),
+#                               version='%s from %s (python %s)'
+#                               % (dist, dist.location, python_version),
                                usage='%prog [paster_options] COMMAND [command_options]')
 
 parser.add_option(
